python/py-jellyfish/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        jamesturk jellyfish 0.8.2
+fetch.type          git
```

Comment:
also here, please use PyPI instead, I think the tarball contains everything that's needed

Reply:
Pypi won't work—you need the C code for the build. See https://github.com/jamesturk/jellyfish and https://github.com/jamesturk/cjellyfish/tree/68951badc8087c8a996e41f4a697abbb560597cd

Reply:
sure you need the C code, but if it's that important you don't think they would include that in the PyPI tarball?

Reply:
I've encountered a fair number of python modules with a non-functional pypi tarball.

Reply:
This one works. Not sure what I did wrong in the original pypi attempt, but uses pypi now.